client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -165,11 +168,16 @@ function HelpSearchResults( {
 					<div className="inline-help__results-cell">
 						<a
 							href={ localizeUrl( link ) }
-							onClick={ ( event ) => {
-								event.preventDefault();
-								selectSearchResult( resultIndex );
-								onLinkClickHandler( event, result );
-							} }
+							onClick={
+								external
+									? null
+									: ( event ) => {
+											event.preventDefault();
+											selectSearchResult( resultIndex );
+											onLinkClickHandler( event, result );
+									  }
```

Comment:
We certainly don't want `.preventDefault()` to be called when `external === true`, but I don't think the logic is quite right yet. We're skipping `selectSearchResult()` and `onLinkClickHandler()`, but `onLinkClickHandler()` is where the `onSelect` prop gets called. That means the `trackResultView()` function over in `<HelpSearch>` is currently never being called.

Maybe the `.preventDefault()` call is the only thing in the click handler that should be skipped?

Reply:
Thank you for the review on this and the guidance you provided! I think that all those issues should be fixed with the latest commit now. :)